pkg/fftypes/blockchainevent.go

```diff
@@ -28,4 +28,6 @@ type BlockchainEvent struct {
 	Info       JSONObject     `json:"info,omitempty"`
 	Timestamp  *FFTime        `json:"timestamp,omitempty"`
 	TX         TransactionRef `json:"tx"`
+	Location   string         `json:"-"`
```

Comment:
Is it possible to avoid these?
e.g. only include them on `blockchain.Event`, rather than `fftypes.BlockchainEvent` which is an external API object.

If we keep them here, they definitely need comments to explain they aren't available.

Reply:
Possibly. Let me try that.